_inc/lib/class.jetpack-password-checker.php

```diff
@@ -0,0 +1,1288 @@
+<?php //phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
FYI @zinigor I added these ignore statements to the new files and added them to the phpcs whitelist since they're new and clean.